src/app/routes/article/getInitialData/index.test.ts

```diff
@@ -11,6 +11,8 @@ process.env.BFF_PATH = 'https://mock-bff-path';
 
 const agent = { cert: 'cert', ca: 'ca', key: 'key' };
 
+const mockGetAgent = () => Promise.resolve(agent as unknown as Agent);
```

Comment:
`agent as unknown as Agent` is this a double cast? (just learning my ts!)

Reply:
Yea just a TS trick to satisfy the tests. The mocked `agent` isn't the full instance of `Agent`, so we cast it to `unknown` first and then cast it to the type we want to type-check.

Reply:
ooh where did you learn that, sneaky!

Reply:
I forgot the name of it, 'double assertion': https://basarat.gitbook.io/typescript/type-system/type-assertion#double-assertion

Not the _best_ practice tbh, but fine to satisfy the compiler for tests.